src/mount.ts

```diff
@@ -73,6 +74,25 @@ function getInstanceOptions(
   return resultOptions
 }
 
+// Standard Class component
```

Comment:
nit: I would remove "Standard" (here and everywhere), as I suspect most people won't write this component like this. I would use "Class component (without vue-class-component)" instead.